src/AppBundle/Entity/User.php

```diff
@@ -260,7 +292,9 @@ public function getCityGeonameId()
      */
     public function setBirthDate($birthDate)
     {
-        $this->birthDate = $birthDate;
+        if ($birthDate) {
```

Comment:
kas tai yra default'inė gimimo data? :D Jeigu birthday yra neprivalomas, tai reikia daryti nullable ir viskas.